spec/operators/map-spec.js

```diff
@@ -88,10 +116,12 @@ describe('Observable.prototype.map()', function () {
     });
 
     expectObservable(r).toBe(expected, values);
+    expectSubscriptions(a.subscriptions).toBe(asubs);
   });
 
   it('should map with index until completed', function () {
     var a = hot('-5-^-4--3---2----1--|--8--|--#');
```

Comment:
(I know this is out of scope for PR) do we allow this kind of marble diagram? maybe this need to be refactored in further too?